invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/administration/userModeration/search/SearchResultItemLayout.js

```diff
@@ -77,7 +77,13 @@ class SearchResultItemComponent extends Component {
         >
           @{splitEmail[1]}
         </Table.Cell>
-        <Table.Cell />
+        <Table.Cell>
+          {/* TEMPORARY */}
+          <a href={`/administration/records?q=parent.access.owned_by.user:${user.id}`}>
```

Comment:
you should always URL encode anything in the param value. Please make sure that you change it (and test it) to `q=encoded(...)`